README.md

```diff
@@ -0,0 +1,73 @@
+# dotfiles
+
+Assalamu Alaikum Wa Rahmatullah. Welcome to my dotfiles repo.
+
+## Process
+
+- First you need to create a directory. 
+```
+mkdir dotfiles
+```
+this ```dotfiles``` directory is our local repo.
+
+- Now clone the dot manager into your ```dotfiles``` directory.
+```
+curl -L https://github.com/ahmubashshir/dotfiles/raw/master/dots > dots
+```
+
+- Now init your local repo.
+```
+git -C dotfiles init
+```
+
+- Change directory & enter into dotfiles directory.
+```
+cd dotfiles
+```
+
+- Now execute the dot manager.
+```
+chmod +x dots
+```
+
+- For easy use you should add this line into your ```.bashrc``` or ```.zshrc``` file.
+```
+alias dots='$HOME/<your dotfile directory>/dots'
+```
+
+> :warning: **Warning**
+>
+> Make sure to change ```<your dotfile directory>```
+
+
+- Now link track dotfiles to your local repo one by one.
+```
+dots cp <dotfile path>
+```
+
+> :warning: **Example**
+>
+> If I want to link my ```.bashrc``` to my local repo, I should run -   
+> ```dots cp ~/.bashrc```
+
+- Now add, commit and push to your github!
+  
+  
+## Usage of dot manager.
+
+> You can find dot manager [here](https://github.com/ahmubashshir/dotfiles/blob/master/dots)
+
+| Command | Work |
```

Comment:
It'd be better to use `task` instead of `work`.

Reply:
done....